c/aws-c-common/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:18.04
+
+RUN apt-get -y update
+RUN apt-get -y upgrade
```

Comment:
Unfortunately, this will cause problems for Docker containers. More info is [here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get), but in general, `upgrade` should be avoided and `update` needs to be in the same `RUN` statement as `install`.